src/machine/uart.go

```diff
@@ -1,11 +1,19 @@
-// +build atmega esp nrf sam sifive stm32 k210 nxp
+// +build atmega esp nrf sam sifive stm32 k210 nxp rp2040
 
 package machine
 
 import "errors"
 
 var errUARTBufferEmpty = errors.New("UART buffer empty")
 
+type UARTParity int
```

Comment:
I think this also needs some documentation.

Reply:
Done.